docs/reference/data-streams/downsampling-manual.asciidoc

```diff
@@ -341,122 +325,260 @@ GET /sample-01*/_search
 
 [discrete]
 [[downsampling-manual-run]]
-==== Run downsampling for the index
+==== Downsample the TSDS
+
+A TSDS can't be downsampled directly. You need to downsample its backing indices
+instead. You can see the backing index for your data stream by running:
+
+[source,console]
+----
+GET /_data_stream/my-data-stream
+----
+// TEST[continued]
+
+This returns:
+
+[source,console-result]
+----
+{
+  "data_streams": [
+    {
+      "name": "my-data-stream",
+      "timestamp_field": {
+        "name": "@timestamp"
+      },
+      "indices": [
+        {
+          "index_name": ".ds-my-data-stream-2023.07.26-000001", <1>
+          "index_uuid": "ltOJGmqgTVm4T-Buoe7Acg"
+        }
+      ],
+      "generation": 1,
+      "status": "GREEN",
+      "template": "my-data-stream-template",
+      "hidden": false,
+      "system": false,
+      "allow_custom_routing": false,
+      "replicated": false,
+      "time_series": {
+        "temporal_ranges": [
+          {
+            "start": "2023-07-26T09:26:42.000Z",
+            "end": "2023-07-26T13:26:42.000Z"
+          }
+        ]
+      }
+    }
+  ]
+}
+----
+// TESTRESPONSE[s/".ds-my-data-stream-2023.07.26-000001"/$body.data_streams.0.indices.0.index_name/]
+// TESTRESPONSE[s/"ltOJGmqgTVm4T-Buoe7Acg"/$body.data_streams.0.indices.0.index_uuid/]
+// TESTRESPONSE[s/"2023-07-26T09:26:42.000Z"/$body.data_streams.0.time_series.temporal_ranges.0.start/]
+// TESTRESPONSE[s/"2023-07-26T13:26:42.000Z"/$body.data_streams.0.time_series.temporal_ranges.0.end/]
+<1> The backing index for this data stream.
+
+Before a backing index can be downsampled, the TSDS needs to be rolled over and
+the old index needs to be made read-only.
 
-Before running downsampling, the index needs to be set to read only mode:
+Roll over the TSDS using the <<indices-rollover-index,rollover API>>:
 
 [source,console]
 ----
-PUT /sample-01/_block/write
+POST /my-data-stream/_rollover/
 ----
 // TEST[continued]
 
-And now, you can use the <<indices-downsample-data-stream,downsample API>> to
-downsample the index, setting the time series interval to one hour:
+Copy the name of the `old_index` from the response. In the following steps,
+replace the index name with that of your `old_index`.
+
+The old index needs to be set to read-only mode. Run the following request:
 
 [source,console]
 ----
-POST /sample-01/_downsample/sample-01-downsample
+PUT /.ds-my-data-stream-2023.07.26-000001/_block/write
+----
+// TEST[skip:We don't know the index name at test time]
+
+Next, use the <<indices-downsample-data-stream,downsample API>> to downsample
```

Comment:
I just wanted to make a note here...after merging this PR https://github.com/elastic/elasticsearch/pull/97557 the name of the downsample index will be known in advance. We made the name generation in such a way to be predictable to support persistent tasks. As a result this docs/tests might need to be changed after we release the new downsampling with persistent tasks support.